packages/graphql/src/translate/translate-top-level-cypher.ts

```diff
@@ -52,10 +52,12 @@ export function translateTopLevelCypher({
     if (annotation) {
         applyAuthentication({ context, annotation });
     }
+    // TODO: is this even possible now?
```

Comment:
What is not possible now? :rofl:

Reply:
because we don't allow `@authorization` directive on top-level cypher fields anymore..